tests/behavior/behavior.rs

```diff
@@ -28,6 +28,7 @@ use futures::TryStreamExt;
 use http::header;
 use isahc::AsyncReadResponseExt;
 use log::debug;
+use opendal::layers::*;
```

Comment:
please list used layers

Reply:
Let's relax the rule in testing code :rofl:

Reply:
> Let's relax the rule in testing code rofl

In testing code it's ok I think (doggy)